.gitignore

```diff
@@ -43,3 +43,9 @@ tests/venv
 .cache
 
 docs/generated/
+
+/core/cache-main
+/core/cache-tests
+/**/.gitignore
```

Comment:
@ewencp with the exception of the `.gitignore` unders tests that I excluded from being ignores, the other `.gitignore`s are generated by Eclipse and the result is that a freshly cloned Kafka source tree after the `./gradlew eclipse` command had run is full of files that git tells you to commit, while it'd be nice if the tree was clean.